gapipy/resources/base.py

```diff
@@ -55,6 +55,9 @@ def __repr__(self):
         else:
             return '<{}: {}>'.format(self.__class__.__name__, self.id)
 
+    def __hash__(self):
+        return hash(self.__repr__())
```

Comment:
I though a resource would always be a stub?

https://github.com/gadventures/gapipy/blob/master/gapipy/models/base.py#L122-L124